pkg/features/antrea_features.go

```diff
@@ -177,6 +178,50 @@ var (
 		AdminNetworkPolicy:          {Default: false, PreRelease: featuregate.Alpha},
 	}
 
+	// AgentGates consists of all known feature gates for the Antrea Agent.
```

Comment:
We need a unit test which will validate that every defined FeatureGate is present in at least one of `AgentGates` or `ControllerGates`. While this doesn't guarantee perfect accuracy of these 2 sets, it will at least prevent contributors from forgetting to update the sets when they define a new FeatureGate.